src/errors/test-run/templates.js

```diff
@@ -87,7 +87,7 @@ export default {
     `),
 
     [TYPE.actionUnsupportedUrlProtocolError]: err => markup(err, `
-        The <code>${err.argumentName}</code> argument specifies a URL that uses an unsupported <code>${err.protocol}</code> protocol. Only HTTP and HTTPS are supported, as well as protocol-relative and relative URLs.
+        The <code>${err.argumentName}</code> argument specifies a URL that uses an unsupported <code>${err.protocol}</code>:// protocol. Only HTTP:// and HTTPS:// are supported, as well as protocol-relative and relative URLs.
```

Comment:
ok, I agree that there are protocols like `mail` or application protocols that may need `://` in the error message to make it clear what we are talking about. But that's hardly the case with HTTP and HTTPS...
```
        The <code>${err.argumentName}</code> argument specifies a URL that uses an unsupported '<code>${err.protocol}</code>://' protocol. Only HTTP and HTTPS are supported, as well as protocol-relative and relative URLs.
```